src/installer.ts

```diff
@@ -170,6 +175,36 @@ export async function getNode(
   core.addPath(toolPath);
 }
 
+function isLts(versionSpec: string): boolean {
+  return versionSpec.startsWith('lts')
+}
+
+function findLtsVersionFromManifest(
+  versionSpec: string,
+  stable: boolean,
+  candidates: Array<tc.IToolRelease & { lts?: string }>
```

Comment:
Could you please create new type for candidates
```
type INodeRelease = tc.IToolRelease & { lts?: string }
```
I think it can be reused somewhere

Reply:
@maxim-lobanov Good point. I'll use `interface` then, to be more consistent with the existing code (which as far as I can see gives preference to interfaces). No objections?

```ts
interface INodeRelease extends tc.IToolRelease {
  lts?: string;
}
```

Reply:
✔️

Reply:
Looks good, thanks!